internal/worker/changestream/doc.go

```diff
@@ -1,11 +1,17 @@
 // Copyright 2025 Canonical Ltd. Licensed under the AGPLv3, see LICENCE file for
 // details.
 //
-// Package changestream defines a worker that manages the lifecycle of the
+// Package changestream makes the databases created by the
```

Comment:
The concept of a database as people would tend to think of it is not exposed. I would say it "adds the ability to create watchers to the transaction runners created..."

Then "These enhanced runners are then...".

Reply:
There are two "create" in your rewrite. Is that a typo?

Reply:
Following our chat exhanged, updated to 

> Package changestream equips the transaction runners created by the internal/worker/dbaccessor package with the ability to watch for specific changes to data.